Bottom: 113f7399e063f97face8e6cc0e203ba1613d414b
Top:    7322136917cf08c1496927383a638f7611d37012
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 03:42:03 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 964e235..863e2ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -392,8 +392,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
